src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const tableBody = document.querySelector('tbody');
+const tableHead = table.querySelector('thead');
+const rowList = [...tableBody.rows];
+
+function getNumber(string) {
+  let str = '';
+
+  for (let i = 0; i <= string.length; i++) {
+    if (!isNaN(+string[i])) {
+      str += string[i];
+    }
+  }
+
+  return +str;
+}
+
+tableHead.addEventListener('click', e => {
+  const headCol = e.target;
+
+  if (!headCol) {
+    return;
+  }
+
+  switch (headCol.innerText) {
+    case 'Name':
+      rowList.sort((a, b) => {
+        const aElem = a.cells[0].innerText;
+        const bElem = b.cells[0].innerText;
```

Comment:
```suggestion
  const index = [...document.querySelectorAll('th')].findIndex(item => item.innerText === headCol.innerText);

  switch (headCol.innerText) {
    case 'Name':
      rowList.sort((a, b) => {
        const aElem = a.cells[index].innerText;
        const bElem = b.cells[index].innerText;
```
Тогда у тебя ничего не сломается, если переставить колонки местами в DOM